test/Twig/Tests/TemplateTest.php

```diff
@@ -262,6 +266,8 @@ public function testGetAttributeStrict($defined, $value, $object, $item, $argume
             } catch (Twig_Error_Runtime $e) {
                 if (null !== $exceptionMessage) {
                     $this->assertSame($exceptionMessage, $e->getMessage());
+                } else {
+                    $this->addToAssertionCount(1);
```

Comment:
much better would be to change this to use ``expectException`` and ``expectExpectionMessage`` (or ``setExpectedException('Twig_Error_Runtime', $exceptionMessage)`` on older versions) and remove the try/catch entirely